CRM/Batch/BAO/Batch.php

```diff
@@ -594,6 +596,7 @@ public static function exportFinancialBatch($batchIds, $exportFormat) {
     else {
       CRM_Core_Error::fatal("Could not locate exporter: $exporterClass");
     }
+    $exporter->_isDownloadFile = $downloadFile;
```

Comment:
oops sorry, Extension had a over-ridden copy of this file therefore i didn't get the error while testing. I have updated the changes now.